dashboard/src/components/TooltipDateTime/TooltipDateTime.tsx

```diff
@@ -0,0 +1,55 @@
+import { format } from 'date-fns';
+
+import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/Tooltip';
+import { getDateOffset } from '@/utils/utils';
+
+type TooltipDateTimeProps = {
+  dateTime: string;
+  dateFormat?: string;
+  timeFormat?: string;
+  lineBreak?: boolean;
+  showLabelTime?: boolean;
+  showLabelTZ?: boolean;
+  showTooltip?: boolean;
+};
+
+const TooltipDateTime = ({
+  dateTime,
+  dateFormat,
+  timeFormat,
+  lineBreak,
+  showLabelTime,
+  showLabelTZ,
+  showTooltip,
+}: TooltipDateTimeProps): JSX.Element => {
+  const dateObj = new Date(dateTime);
```

Comment:
not all strings are valid datetime, so please validate the string and if it is not a valid datetime, return something like `<div> - </div>`

Reply:
if you do this you can make it accept `dateTime?:string` again since there will be an internal validation